ale_linters/go/goflycheck.vim

```diff
@@ -0,0 +1,9 @@
+" Author: dzhou121 <dzhou121@gmail.com>
+" Description: goflycheck for Go files
+
+call ale#linter#Define('go', {
+\   'name': 'goflycheck',
+\   'executable': 'goflycheck',
+\   'command': 'goflycheck %s',
```

Comment:
If you're checking the file on disk, you should add the `'lint_file': 1` flag below. Otherwise the linter will run while you're typing.